x-pack/legacy/plugins/snapshot_restore/public/app/sections/home/snapshot_list/snapshot_details/snapshot_details.tsx

```diff
@@ -199,6 +199,18 @@ export const SnapshotDetails: React.FunctionComponent<Props> = ({
                             onSnapshotDeleted
                           )
                         }
+                        isDisabled={snapshotDetails.isManagedRepository}
+                        title={
+                          snapshotDetails.isManagedRepository
+                            ? i18n.translate(
+                                'xpack.snapshotRestore.snapshotDetails.deleteManagedRepositorySnapshotButtonTitle',
+                                {
+                                  defaultMessage:
+                                    'You cannot delete a managed repository snapshot.',
```

Comment:
I defer to @gchaps on this one but I think this wording is a bit ambiguous, because it's not clear if "repository snapshot" is the object qualified by the word "managed". How about rephrasing this to "You cannot delete a snapshot stored in a managed repository."

This comment applies to similar language elsewhere in the PR.

Reply:
+1 to "You cannot delete a snapshot stored in a managed repository."